var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -123,13 +125,17 @@ class NetcdfC(AutotoolsPackage):
     # https://github.com/Unidata/netcdf-c/issues/250
     depends_on("hdf5@:1.8", when="@:4.4.0")
 
+    depends_on("zstd", when="+zstd")
+
     # The feature was introduced in version 4.1.2
     # and was removed in version 4.4.0
     # conflicts('+cdmremote', when='@:4.1.1,4.4:')
 
     # The features were introduced in version 4.1.0
     conflicts("+parallel-netcdf", when="@:4.0")
     conflicts("+hdf4", when="@:4.0")
+    # zstd compression was introdcued in 4.9.0
+    conflicts("+zstd", when="@:4.8")
```

Comment:
```suggestion
```

Reply:
@kgerheiser please, keep the authorship of the suggested changes